app/jobs/session_store_truncate_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SessionStoreTruncateJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    # Copying from: rake db:sessions:trim
+    cutoff_period = (ENV["SESSION_DAYS_TRIM_THRESHOLD"] || 30).to_i.days.ago
```

Comment:
Can we put this in settings.yml and use `Settings.session_expiry_period` or something?

Reply:
Will add that now